gravitee-am-gateway/gravitee-am-gateway-handler/gravitee-am-gateway-handler-oidc/src/main/java/io/gravitee/am/gateway/handler/oidc/service/discovery/impl/OpenIDDiscoveryServiceImpl.java

```diff
@@ -173,6 +174,14 @@ public OpenIDProviderMetadata getConfiguration(String basePath) {
         final String clientAuth = env.getProperty("http.ssl.clientAuth", String.class, "none");
         final String clientCert = env.getProperty(ConstantKeys.HTTP_SSL_CERTIFICATE_HEADER, String.class);
 
+        if (domain.useCiba()) {
+            openIDProviderMetadata.setBackchannelAuthenticationSigningAlg(JWAlgorithmUtils.getSupportedBackchannelAuthenticationSigningAl());
+            openIDProviderMetadata.setBackchannelTokenDeliveryModesSupported(List.of(CIBADeliveryMode.POLL));
+            openIDProviderMetadata.setBackchannelAuthenticationEndpoint(getEndpointAbsoluteURL(basePath, CIBA_AUTHENTICATION_ENDPOINT));
+            openIDProviderMetadata.setBackchannelUserCodeSupported(false);
+        }
+
+        if (secured && !clientAuth.equalsIgnoreCase("none")) {
```

Comment:
```
final boolean mtlsEnabled = clientCert != null || (secured && !clientAuth.equalsIgnoreCase("none"));
if (mtlsEnabled) {
---
}
```

To avoid potential regression.